From: Andrey Borzenkov <arvidjaar@gmail.com>
Date: Fri, 21 Feb 2014 20:44:06 +0400
Subject: man: fix description of systemctl --after/--before

It was backward - --after fetches After property, so units shown really
come *before* unit given as argument. Same for --before.

(cherry picked from commit 4a77ca700d1841c2b8a86ed5b7f3495db61c236d)
(cherry picked from commit ba7a578a5d2789e86f046c99edec6d517a2c2fcb)
---
 man/systemctl.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemctl.xml b/man/systemctl.xml
index d4048a88..03f2bc5e 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -176,8 +176,8 @@ systemctl start foo
         <term><option>--before</option></term>
 
         <listitem>
-          <para>Show which units are started after or before
-          with <command>list-dependencies</command>, respectively.
+          <para>Show after (before) which units the specified unit is started
+          with <command>list-dependencies</command>.
           </para>
         </listitem>
       </varlistentry>
